frontend/ui/widget/dictquicklookup.lua

```diff
@@ -429,7 +430,14 @@ function DictQuickLookup:update()
                 {
                     -- if dictionary result, do the same search on wikipedia
                     -- if already wiki, get the full page for the current result
-                    text = self.is_wiki and _("Wikipedia full") or _("Wikipedia"),
+                    text_func = function()
+                        if self.is_wiki then
+                            -- @translators Full Wikipedia article.
```

Comment:
This actually attaches itself to the first extracted string if you leave the line as is, but I figured it was better to be abundantly clear about it.